.drone.starlark

```diff
@@ -477,6 +477,11 @@ def javascript():
 				'pull': 'always',
 				'environment': params['extraEnvironment'],
 				'commands': params['extraCommandsBeforeTestRun'] + [
+					'apt update -y',
+					'apt-get install xvfb libxrender1 libxtst6 libxi6 -y',
+					'/usr/bin/Xvfb :99 &',
+					'export DISPLAY=:99.0',
+					'apt-get install firefox -y',
```

Comment:
These commands should be able to go in the `javascript` section of `config` up the top, see an example in `files_antivirus` `.drone.star` (where `extraCommandsBeforeTestRun` is used for `phpunit`)

Then we do not have different starlark code in this repo.